static/app/views/dashboardsV2/widgetCard/index.tsx

```diff
@@ -234,6 +235,17 @@ class WidgetCard extends Component<Props, State> {
         showIncompleteDataAlert = prior < METRICS_BACKED_SESSIONS_START_DATE;
       }
     }
+    if (widget.displayType === DisplayType.TOP_N) {
+      const queries = widget.queries.map(query => ({
+        ...query,
+        // Use the last aggregate because that's where the y-axis is stored
+        aggregates: query.aggregates.length
+          ? [query.aggregates[query.aggregates.length - 1]]
+          : [],
+      }));
+      widget.queries = queries;
+      widget.limit = DEFAULT_RESULTS_LIMIT;
+    }
```

Comment:
Doing this so we don't have to pass DispalyType to transform series in widget queries. Also this isn't dataset specific.